static/app/components/organizations/projectSelector/index.tsx

```diff
@@ -65,10 +64,6 @@ type Props = {
    * Represents if a search is taking place
    */
   searching?: boolean;
-  /**
-   * Show the pin button in the dropdown's header actions
-   */
-  showPin?: boolean;
```

Comment:
we might still need to maintain this prop until we completely remove the `selection-filters-v2` feature flag, or else for ST and self-hosted sentry users the pin/lock will show for the global header project selector which still exists until we completely remove the feature flag.

Reply:
true! added it back in https://github.com/getsentry/sentry/pull/34503/commits/aa9613937bf7097981e38926edd815ec5ef8f1d8

Reply:
dang so quick 💯